### PR TITLE
Event endpoint implementation

### LINKED ISSUE #7: Create event/:id/join endpoint
```
ENDPOINT
POST /api/v1/event/:id/join

RESPONSE
	success: bool
	error?: string
```

### LINKED ISSUE #9: List events endpoint
```
ENDPOINT
POST /api/v1/events/

REQUEST
	title: string
	description: string
	start_date: string (yyyy-MM-dd HH:mm)
	end_date: string (yyyy-MM-dd HH:mm)

RESPONSE
	success: bool
	error?: string
```

### PR DESCRIPTION
**Pull request by emilio2hd**
Refactoring the way that api expose data through serializer to include pagination with kaminari
Rename event's date fields, because 'end' is a reserved word
Implement the event requirement
Close #7, close #9 

New gem added:
- [kaminari](https://github.com/amatsuda/kaminari) paginator for Ruby webapps

### CODE REVIEW

File: config/initializers/kaminari_config.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+  # config.default_per_page = 25
```
Comment (alaphao) [inline config/initializers/kaminari_config.rb]: These are the default (and current) values?

File: config/initializers/kaminari_config.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  # config.max_per_page = nil
+  # config.window = 4
+  # config.outer_window = 0
+  # config.left = 0
+  # config.right = 0
+  # config.page_method_name = :page
+  # config.param_name = :page
+end
```
Comment (alaphao) [inline config/initializers/kaminari_config.rb]: These are the default (and current) values for the pagination?

Comment (emilio2hd) [inline config/initializers/kaminari_config.rb]: Yes, they are.

**Comment by emilio2hd** [comment]
Thanks @alaphao :smile:

**Pull request merged**